services/backend-for-frontend-deno/src/tracing.ts

```diff
@@ -0,0 +1,25 @@
+import { NodeSDK } from '@opentelemetry/sdk-node';
```

Comment:
This is not needed in Deno at all.

Reply:
You just have to start Deno with `--unstable-otel` and `OTEL_DENO=true` and it all just works OOTB.